### DISCUSSION TITLE
Random flux sampling with min_growth

### DISCUSSION THREAD

**Discussion by jmol0917** [micom-dev/micom#129] [Q&A]
Hi!

I am modelling a couple of community metabolic states using min_growth like this:
```
coo_solution = community.cooperative_tradeoff(min_growth=[1.1, 0.1, 6.45], fraction=1.0, fluxes=True, pfba=True)
```

Any advice in terms of running random flux sampling while constraining growth like so and using cooperative_tradeoff? Would this even make any sense or is the solution from cooperative_tradeoff "unique" when it comes to fluxes plus growth tradeoffs?

I imagine that I could specify min_growth values as lower bounds for the biomass reaction of each of the corresponding members of the community, and then run something like ```cobra.sampling.sample()``` but that would generate solutions without considering the implications around cooperative_tradeoff?

Cheers!

**Comment by cdiener** [comment]
Apologies for the late reply. Yep, that would make sense, so sampling will probably be pretty slow with large models. You could constrain the growth rates with what you got from cooperative tradeoff and then sample from that. Unfortunately, `cobra.sampling.sample` does not work with MICOM models until I fix #72.
